src/styles/index.scss

```diff
@@ -1,3 +1,109 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    display: flex;
+    justify-content: center;
+    margin-left: 20px;
+  }
+
+  &__name {
+    display: flex;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    justify-content: center;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__product {
+    &--code {
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: #616070;
+      display: flex;
+      margin-left: 20px;
+      margin-top: 4px;
+      margin-bottom: 16px;
+      justify-content: space-between;
+    }
+  }
+
+  &__rate {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &--star4:nth-child(-n + 5) {
+      background-image: url(/src/images/star-active.svg);
+    }
+
+    &--star:last-child {
+      margin-right: 0;
+    }
+
+    &--star5 {
+      display: flex;
+    }
+
+    &--review {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+      text-align: center;
+    }
+  }
+
+  &__price {
+    justify-content: space-between;
+    display: flex;
+    width: 166px;
+
+    &--item {
+      color: #616070;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &--value {
+      color: #060b35;
+      font-weight: 700;
+    }
+  }
+
+  &__card__button {
```

Comment:
The class name `&__card__button` seems incorrect according to BEM naming conventions. It should be `&__button` since it is a modifier of the `card` block.